### DISCUSSION TITLE
Heroku login error

### DISCUSSION THREAD

**Discussion by VitaliyBobyk** [kevinzg/facebook-scraper#325] [Q&A] [answered]
Hi, using library to get posts from group, and have a issue, that locally, can do it even without credentials option in get_posts method, but on heroku, with or without credentials options, not working.

`File "/app/.heroku/python/lib/python3.9/site-packages/facebook_scraper/facebook_scraper.py", 
line 329, in login raise exceptions.LoginError("Login unsuccessful")
facebook_scraper.exceptions.LoginError: Login unsuccessful`

**Comment by neon-ninja** [comment, marked as answer]
It's possible that too many people have scraped facebook from heroku IP ranges, which cause facebook servers to be more suspicious of heroku IP addresses. Try pass cookies instead of credentials.
